### ISSUE TITLE
Cannot write a Dockerfile without a Git remote. 

### ISSUE THREAD

**Issue by awconway** [karthik/binder-test#2]
I tried the holepunch functions and got this error from the write_deockerfile() function:

> Error: Cannot write a Dockerfile without a Git remote. Connect this to a git remote before generating a Dockerfile

So I tried this binder-test following your instructions in the readme but got the exact same error. I ran git remote -v to check both times and there is indeed the git remote where it should be for fetch and push.

Any suggestions?

**Comment by karthik** [comment]
@awconway Thanks so much for this report. Can you tell me what version of holepunch you have?  I'm using

```r
holepunch    * 0.1.11.9000 2019-06-21 [1] local     
```

and I just followed these instructions myself and it worked fine. Can you run this and see what it reports right before `write_dockerfile()` throws that error.

```r
holepunch:::has_a_git_remote()
```

**Comment by awconway** [comment]
Thanks for the reply. I just installed holepunch yesterday so have version holepunch_0.1.11.9000.

> holepunch:::has_a_git_remote()
[1] FALSE

But from the terminal:

Aarons-Mac-mini:testholepunch aaronconway$ git remote -v
origin  https://github.com/awconway/testholepunch.git (fetch)
origin  https://github.com/awconway/testholepunch.git (push)

**Comment by awconway** [comment]
I just did the binder-test on my server that is running Rstudio and it works.  I'll just close this issue because it's more likely a problem to do with my mac setup rather than the package. Sorry about that.

**Comment by JKRWard** [comment]
Hey @karthik 

I had the same error message yesterday trying to run through the tutorial using 

```
holepunch_0.1.14.9000  
```
```r
holepunch:::has_a_git_remote()
[1] FALSE
```
Having traced it back it seems to be because I had `usethis` 1.4.0 and `git_remotes` was first introduced  1.5.0  so although it's returning FALSE because an error is generated that's because the function is missing not because the remote is missing 😺

**Comment by karthik** [comment]
Awesome, thanks @JKRWard! I really appreciate this. I can add a minimum version to usethis in a fix. 🙏

**Comment by karthik** [comment]
I pushed a fix to require usethis > 1.4. I hope this will resolve the error but I'll keep an eye on it.

**Comment by RebeccaPPayne** [comment]
Hi Im having this exact same problem. Any further suggestions?

Error: Cannot write a Dockerfile without a Git remote. Connect this to a git remote before generating a Dockerfile
